jmclient/jmclient/storage.py

```diff
@@ -323,7 +323,7 @@ class VolatileStorage(Storage):
 
     def __init__(self, password=None, data=None):
         self.file_data = None
-        super(VolatileStorage, self).__init__('VOLATILE', password,
+        super().__init__('VOLATILE', password,
                                               create=True)
```

Comment:
some whitespace issues, also at other places

Reply:
Wops! Just added a commit to fix these.